draft-ietf-dhc-addr-notification.md

```diff
@@ -251,7 +251,18 @@ The ADDR-REG-REPLY message only indicates that the ADDR-REG-INFORM message has b
 
 ## Registration Expiry and Refresh
 
-The client MUST refresh the registration every AddrRegRefresh seconds, where  AddrRegRefresh is min(1/3 of the Valid Lifetime filed in the very first PIO received to form the address; 4 hours). Registration refresh packets SHOULD be retransmitted using the same logic as described in the 'Retransmission' section below. In particular, retransmissions SHOULD be jittered to avoid synchronization causing a large number of registrations to expire at the same time.
+The client MUST refresh addresses as described below. Each refresh is scheduled after AddrRegRefresh seconds, where AddrRegRefresh is min(4 hours, 80% of the address's current Valid Lifetime). Refreshes SHOULD be jittered by +/- 10% to avoid synchronization causing a large number of registrations to expire at the same time.
+
+Whenever the client creates an address or receives a PIO which updates the Valid Lifetime of an existing address, then:
```

Comment:
Might want to say "changes" instead of "updates". Rationale: if the lifetime is currently 597, and we receive an RA with a lifetime of 597, that might be construed as an "update".

Further, we might want to say something like, "changes the Valid Lifetime of an existing address by more than 5%" or "by more than 1 minute" or something like that.

That ensures that if the RAs are slightly desynchronized with the client time, the client does not run this code every time. An example of how this might happen is:
- Server sends an RA with a lifetime of 600, but it is delayed by 300ms by congestion.
- After 60 seconds, server sends an RA with a lifetime of 540, which is not delayed by congestion. At this point the client will think the lifetime is still 541 seconds, and thus it might think that the lifetime "changed".

Reply:
Maybe 5% and 1 minute are too high, but you get the idea. We could say "by 1% (to avoid rescheduling due to clock skew or transmission delays)". No need to do that in this PR though.